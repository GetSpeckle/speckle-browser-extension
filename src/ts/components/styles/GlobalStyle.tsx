import { createGlobalStyle } from 'styled-components'

const GlobalStyle = createGlobalStyle`
	body {
		box-sizing: border-box;
		display: block;
		margin: 0;
	}
`

export default GlobalStyle
