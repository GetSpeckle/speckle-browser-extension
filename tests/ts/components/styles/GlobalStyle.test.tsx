import renderer from 'react-test-renderer';
import GlobalStyle from "../../../../src/ts/components/styles/GlobalStyle";

describe('Testing Global Style', () => {

    test('GlobalStyle Snapshot.', () => {
        //@ts-ignore
        const wow = renderer.create(GlobalStyle.globalStyle.rules).toJSON();
        expect(wow).toMatchSnapshot();
    });
});