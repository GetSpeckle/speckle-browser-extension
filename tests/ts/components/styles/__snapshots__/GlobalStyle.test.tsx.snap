// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Global Style GlobalStyle Snapshot. 1`] = `
"
	body {
		box-sizing: border-box;
		display: block;
		margin: 0;
	}
"
`;
